test/acceptance/features/supportExistingOperatorBackedServices.feature

```diff
@@ -169,18 +169,17 @@ Feature: Support a number of existing operator-backed services out of the box
     And Application can connect to the projected Postgres database
 
 
-  @crdv1beta1
   Scenario: Bind test application to Mysql provisioned by Percona Mysql operator
     Given Percona Mysql operator is running
     * Generic test application is running
     * The Custom Resource is present
           """
-          apiVersion: pxc.percona.com/v1-8-0
+          apiVersion: pxc.percona.com/v1-10-0
```

Comment:
I don't see annotation for this API version in the registry.

Reply:
it is now... I added it